megamek/src/megamek/ai/dataset/UnitAttackAction.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017-2025 The MegaMek Team. All Rights Reserved.
+ *
+ * This file is part of MegaMek.
```

Comment:
If a new-ish file, correct the copyright to correspond to when it was first merged.